Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -303,19 +303,9 @@ public bool TryGetDistanceToNearestSurface(out float distance)
         {
             using (TryGetDistanceToNearestSurfacePerfMarker.Auto())
             {
-                var focusProvider = CoreServices.InputSystem?.FocusProvider;
-                if (focusProvider != null)
-                {
-                    FocusDetails focusDetails;
-                    if (focusProvider.TryGetFocusDetails(this, out focusDetails))
-                    {
-                        distance = focusDetails.RayDistance;
-                        return true;
-                    }
-                }
-
-                distance = 0.0f;
-                return false;
+                // Leaving it like this because currently the sphere pointer does not track the closest grabbable object.
```

Comment:
I'm not sure that a distance of 0 is expected here, especially if it used to return a different result. it seems previously it would return the RayDistance from this pointer to the object it was focusing. At least make a comment and a release note, so people are not surprised, and note you may get bug reports about this in the future since this is a change in the behavior.